Lib/http/server.py

```diff
@@ -1306,7 +1307,7 @@ def _get_best_family(*address):
 
 def test(HandlerClass=BaseHTTPRequestHandler,
          ServerClass=ThreadingHTTPServer,
-         protocol="HTTP/1.0", port=8000, bind=None,
+         protocol="HTTP/1.0", port=8000, bind=None, directory=None,
```

Comment:
as I already wrote, you shouldn't make changes to the module itself that affect its behavior
```suggestion
         protocol="HTTP/1.0", port=8000, bind=None,
```